payloads/library/exfiltration/mysql_dump/payload.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+disk=$(lsblk -fs | grep BashBunny | awk '{print $1}')
```

Comment:
Please add instructions to your readme to indicate to the user that if they aren't used the default drive label that it will need to be changed in the payload.sh script.